charts/kyuubi/templates/kyuubi-configmap.yaml

```diff
@@ -19,12 +19,26 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
   labels:
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
     app.kubernetes.io/name: {{ .Chart.Name }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/version: {{ .Values.image.tag | default .Chart.AppVersion | quote }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+    my-custom-label: my-custom-value   # adding a custom label
```

Comment:
the example stuffs should not be added directly, if you want, please define variables in values.yaml, and use template to inject them.

Reply:
Okay I will define this variable, and  then use it in the  templates by referencing it .